extension.driver.php

```diff
@@ -114,6 +114,20 @@ public function update($previousVersion = false)
              );
         }
 
+        if (version_compare($previousVersion, '2.0.2', '<')) {
```

Comment:
Extension update version needs to be v2.0.2+ for this condition.

Reply:
Can you add the missing lines in the meta.xml file to create the new version please ?

Reply:
@nitriques sure, like this 470fa5b ?

Reply:
Yup!